src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -79,7 +80,10 @@ func (mod *mgmtModule) HandleCall(session *drpc.Session, method drpc.Method, req
 	case drpc.MethodGetAttachInfo:
 		return mod.handleGetAttachInfo(req, cred.Pid)
 	case drpc.MethodDisconnect:
-		return mod.handleDisconnect(req, cred.Pid)
+		// There isn't anything we can do here if this fails so just
+		// call the disconnect handler and return success.
```

Comment:
comment probably isn't necessary anymore since handleDisconnect returns nothing

Reply:
I figure we could keep it since the return nil,nil right underneath it might seem odd without it.

Reply:
My thought was a function that returns nothing can't fail anyway. I'm not that picky about it though.